### DISCUSSION TITLE
Intracrystalline Misorientation Axes

### DISCUSSION THREAD

**Discussion by beroshcr** [mtex-toolbox/mtex#1504] [Ask Anything]
Is there a function on MTEX that allows the plotting of intracrystalline misorientation axes for a grain? I am trying to figure out the most prominent slip-system in the pyroxene grains of my sample.

Thanks!

**Comment by kilir** [comment]
Hi,
in the most general sense `plot(h)` where h is a crystal direction or specimen direction, depending to which misorientation axes you refer.

If you refer rather to how to derive those axes, it depends what you mean: misorientations associated with low angle boundaries, with neighboring pixels, with respect to a reference orientation within a grain. A good place to start would be here: https://mtex-toolbox.github.io/Documentation.html in the sections on grain boundaries or orientations parameters in grains.

Cheers,
Rüdiger
